README.md

```diff
@@ -1 +1,195 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 "Game Market 🎮" project <a name="about-project"></a>
+
+![mobile 1](https://github.com/Fombi-Favour/metrics-capstone/assets/77404317/db6830ff-94dd-45e1-be3a-9ccc7d6d0909)
+![mobile 2](https://github.com/Fombi-Favour/metrics-capstone/assets/77404317/8d4f1667-0ec2-453a-96ab-365a5ae9be81)
+
+**"Game Market" project** is a responsive e-commerce website that displays information of games of different categories. All of these information is fetch from this API
+
+- [Batman Category](https://www.cheapshark.com/api/1.0/games?title=batman)
+- [FIFA Category](https://www.cheapshark.com/api/1.0/games?title=fifa)
+- [WWE Category](https://www.cheapshark.com/api/1.0/games?title=wwe)
+- [Action Category](https://www.cheapshark.com/api/1.0/games?title=action)
+- [NBA Category](https://www.cheapshark.com/api/1.0/games?title=nba)
+- [F1 Category](https://www.cheapshark.com/api/1.0/games?title=f1)
+- [Need For Speed Category](https://www.cheapshark.com/api/1.0/games?title=nfs)
+- [LEGO Category](https://www.cheapshark.com/api/1.0/games?title=lego)
+- [Marvel Category](https://www.cheapshark.com/api/1.0/games?title=marvels)
+- [Call of Duty Category](https://www.cheapshark.com/api/1.0/games?title=call-of-duty)
+- [Mortal Kombat Category](https://www.cheapshark.com/api/1.0/games?title=mortal-kombat)
+- [Devil May Cry Category](https://www.cheapshark.com/api/1.0/games?title=devil-may-cry)
+- [Hitman Category](https://www.cheapshark.com/api/1.0/games?title=hitman)
+- [Street Fighter Category](https://www.cheapshark.com/api/1.0/games?title=street-fighter)
+- [Tekken Category](https://www.cheapshark.com/api/1.0/games?title=tekken)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Version Control System</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://react.dev/">React JS</a></li>
+    <li><a href="https://tailwindcss.com/">Tailwind CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Library</summary>
+  <ul>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Beautiful layouts**
+- **Cool display of the categories and their respective games information**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link. Click here!](https://game-mar23.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Presentation <a name="live-demo"></a>
+
+- Check out the presentation for this project [Click here!](https://www.loom.com/share/d66523b5ec07413c8b23c4582bd99e7f?sid=5147dce9-b707-4f57-8c9c-70db11e177b4)
+
```

Comment:
- You have done great so far :+1: Your project looks great and works as expected. :+1: but there are a few changes you need to make before moving to the next milestone. :+1: :+1: 

- [ ] The presentation video you have seems to be for a different project. The website in your video is not the same as the one here. I'll suggest you add the correct video or you make a new video following the presentation guideline. :+1: :wave: